### PR TITLE
ASU-1429 | Email notification after sending installments to SAP

### PR DESCRIPTION
**Pull request by quyenlq**
- Add a simple email template
- Email will be sent right after the cron task sending installments to SAP
- Fixed a test related to marking sent installments
- Remove unused config for `django-mailer` which we are not using at the moment

### CODE REVIEW

File: invoicing/services.py [hunk 1]
```diff
@@ -12,31 +15,50 @@
...
+    return email.send()
+
+
+def generate_sap_xml_filename(ts: datetime) -> str:
+    return settings.SAP_SFTP_FILENAME_PREFIX + f"{ts.strftime('%Y%m%d%H%M%S')}" + ".xml"
 
 
-def send_xml_to_sap(xml: bytes, filename: str = None) -> None:
+def send_xml_to_sap(
+    xml: bytes, filename: str = None, ts: datetime = timezone.now()
```
Comment (tuomas777) [inline invoicing/services.py:58]: Here `timezone.now()` will be evaluated only once, so the default value will be the same for every call of `send_xml_to_sap()`, which is probably not what we want 🙂

File: apartment_application_service/settings.py [hunk 2]
```diff
@@ -362,6 +361,9 @@
 METADATA_HITAS_PROCESS_NUMBER = env.str("METADATA_HITAS_PROCESS_NUMBER")
 METADATA_HASO_PROCESS_NUMBER = env.str("METADATA_HASO_PROCESS_NUMBER")
 
+TALPA_EMAIL = env.str("TALPA_EMAIL")
+TALPA_EMAIL_REPY_TO = env.str("TALPA_EMAIL_REPLY_TO")
```
Comment (tuomas777) [inline apartment_application_service/settings.py:365]:
```suggestion
TALPA_EMAIL_REPLY_TO = env.str("TALPA_EMAIL_REPLY_TO")
```
Typo 🙂

File: invoicing/management/commands/send_installments_to_sap.py [hunk 3]
```diff
@@ -11,5 +15,10 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         logger.info("Sending installments to SAP...")
-        num_of_installments = send_needed_installments_to_sap()
+        num_of_installments, ts = send_needed_installments_to_sap()
```
[the remark this answers is not in the document]
Comment (quyenlq) [reply invoicing/management/commands/send_installments_to_sap.py]: Agree! Fixed that

**Review by tuomas777 — APPROVED** [review]
LGTM 👍 🐳

**Pull request merged**